foam_controlwrapper/examples/poiseuille.py

```diff
@@ -3,12 +3,12 @@
 """
 
 from simphony.core.cuba import CUBA
```

Comment:
@khiltunen, @jmarcelogimenez ,    the ``foam_controlwrapper/examples/poiseuille.py`` and ``foam_internalwrapper/examples/poiseuille.py`` are also mostly the same.    Both types of wrappers support the same functionality, right?  When looking at the files, it seems so..except for one minor thing (i.e. `CUBAExt.NUMBER_OF_CORES`).  

I would think that these are examples of using simphony-openfoam (regardless of its its the INTERNAL or FILE-IO version). Therefore,  ``foam_controlwrapper/examples/poiseuille.py`` and ``foam_internalwrapper/examples/poiseuille.py` could be replaced by ``examples/poiseuille.py``.  

I would suggest not trying to tackle this in this PR but to address it in a seperate PR and an issue should be opened up now (e.g. "make a single poiseuille.py")